### DISCUSSION TITLE
`Space-k` exists in documentation but not in the editor.

### DISCUSSION THREAD

**Discussion by txtyash** [helix-editor/helix#867] [General] [answered]
There's a feature to look up the help for the word under the cursor(Space-k). It is not actually present inside the editor but only in the documentation and please explicitly state in the documentation that `space` has to be pressed to enter space mode.

**Comment by archseer** [comment, marked as answer]
It's `K` in 0.4.1 and was changed to `space+k` in master. A new release will go out soon to fix the discrepancy.
